bin/rhsm-debug

```diff
@@ -62,20 +62,21 @@ def main():
 if __name__ == '__main__':
     try:
         sys.exit(abs(main() or 0))
-    except SystemExit, e:
-        #this is a non-exceptional exception thrown by Python 2.4, just
-        #re-raise, bypassing handle_exception
+    except SystemExit as sys_err:
+        # This is a non-exceptional exception thrown by Python 2.4, just
+        # re-raise, bypassing handle_exception
         try:
             sys.stdout.flush()
+            sys.stderr.flush()
```

Comment:
@kahowell The rhsm-debug is already addressed. I only added flushing of stderr as bonus.